markdown/ch8.markdown

```diff
@@ -279,7 +274,7 @@ During DNA sequencing some bases are incorrectly read out, and must be fixed, or
 One correction strategy is to find similar reads in your dataset and fix the error by grabbing the correct information from those reads. Or alternatively, you may choose to completely discard those reads containing errors.
 
 However, this is a very inefficient way to do it, because finding similar reads means you would compare each read to every other read.
-This takes $N^2$ operations, or $9 \times 10^14$ for a 30 million read dataset!
+This takes $N^2$ operations, or $9 \times 10^{14}$ for a 30 million read dataset!
```

Comment:
@hdashnow https://github.com/elegant-scipy/elegant-scipy/commits/master it's already been merged, so feel free to rebase when you can! =) I think you probably accidentally sprouted from your ch8 branch, instead of from master.